src/include/traffic_cop/traffic_cop.h

```diff
@@ -115,6 +115,26 @@ class TrafficCop {
 
   bool is_queuing_;
```

Comment:
Can you provide documentation and not make these members public? Add Get/Set methods.